sorc/ncep_post.fd/CALRAD_WCLOUD_newcrtm.f

```diff
@@ -2078,6 +2116,30 @@ SUBROUTINE CALRAD_WCLOUD
                    endif
                  enddo ! channel loop
               end if  ! end of outputting goes 17
+!             Wm Lewis updated idx for g18 on 3 JUN 2022
+              if (isis=='abi_g18')then  ! writing goes 16 to grib
```

Comment:
@EricJames-NOAA I would assume the comments for "writing goes 18"?

Reply:
@WenMeng-NOAA good catch!  I just corrected that, and an earlier incorrect comment for writing goes 17.